packages/analytics/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -9,26 +9,28 @@ environment:
   sdk: ">=2.17.0 <3.0.0"
   flutter: ">=3.0.0"
 
+# Helps `pana` since we do not use Flutter plugins for most platforms
+platforms:
+  ios:
+  android:
+  macos:
+  windows:
+  linux:
+  web:
+
 dependencies:
-  amplify_analytics_pinpoint_android: ">=1.0.0-next.0 <1.0.0-next.1"
-  amplify_analytics_pinpoint_ios: ">=1.0.0-next.0 <1.0.0-next.1"
-  amplify_core: ">=1.0.0-next.0+2 <1.0.0-next.1"
-  aws_common: ">=0.2.4 <0.3.0"
+  amplify_analytics_pinpoint_dart: ^0.1.0
+  amplify_core: ">=1.0.0-next.0+1 <1.0.0-next.1"
```

Comment:
question: Are we sure these deps are correct here from last rebase? seems like other packages get core from .0+2, also noticing the ways of specifying below deps like amplify_secure_storage different than auth.

Reply:
They will be updated in version bump - no worries if they're stale here.